src/modules/UI/scenes/Request/Request.ui.js

```diff
@@ -6,7 +6,8 @@ import {
   Alert,
```

Comment:
This PR is for the FlipInput. What changes were needed in this file for the FlipInput animation?